src/index.html

```diff
@@ -4,9 +4,362 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          class="logo__img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item"><a href="#" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+  
+    <main class="catalog">
+      <div class="catalog__container">
+        <article class="card catalog__card" data-qa="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="computer monitor"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__subtitle">Product code: 195434</p>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <span class="card__reviews">Reviews: 5</span>
+          </div>
+
+          <div class="card__price">
+            <h3 class="card__price-title">Price:</h3>
+
+            <p class="card__price-value">$2,199</p>
+          </div>
+
+          <a
+            class="card__button"
+            data-qa="card-hover"
+            href="#"
+          >
+            BUY
+          </a>
+        </article>
+        <article class="card catalog__card">
```

Comment:
```suggestion
        </article>
        
        <article class="card catalog__card">
```

Well structure , I suggest  add empty lines between multi line sibling blocks of HTML.

Reply:
updated the html file